redistimeseries/client.py

```diff
@@ -30,7 +30,7 @@ def list_to_dict(aList):
                 for i in range(len(aList))}
 
 def parse_range(response):
-    return [tuple((l[0], l[1].decode())) for l in response]
+    return [tuple((l[0], float(l[1]))) for l in response]
```

Comment:
@ashtul did you test it on both python versions? there's a known caveat with Python3 and redis-py, if you don't pass a specific argument it return a byte array instead of string. `float()` might know how to handle bytearrays but im not 100% sure about that.

Reply:
@danni-m 
It wasn't tested before on python3 but I have just tested and it passed.

Reply:
@danni-m ???